docs/sp/site-users.md

```diff
@@ -82,9 +82,9 @@ import "@pnp/sp/webs";
 import "@pnp/sp/site-users/web";
 
 const user = await sp.web.ensureUser("userLoginname")
-const users = await sp.web.siteUsers;
+const users = await sp.web.siteUsers();
   
-await users.add(user.data.LoginName);
```

Comment:
I think this one is right? `users` in this case is an instance of the siteUsers collection, purposefully not executed - and then used to add a user.

Reply:
Oh, you are definitely right. I'll fix that and change it back!